doc/en/example/simple.rst

```diff
@@ -819,7 +819,7 @@ case we just write some information out to a ``failures`` file:
             with open("failures", mode, encoding="utf-8") as f:
                 # let's also access a fixture for the fun of it
                 if "tmp_path" in item.fixturenames:
-                    extra = " ({})".format(item.funcargs["tmp_path"])
+                    extra = " ({})".format(item._request.getfixturevalue("tmp_path"))
```

Comment:
We shouldn't encourage access to a private attribute, so let's instead do

```py
if "tmp_path" in item.funcargs:
    extra = " ({})".format(item.funcargs["tmp_path"])
```

It not the same but good enough for this example.